pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -365,13 +440,25 @@ func updateNodesStatus(c client.Client, instance *computenodev1alpha1.ComputeNod
 	var readyWorkers int32 = 0
 	nodesStatus := []computenodev1alpha1.Node{}
 	for _, node := range nodeList.Items {
+		log.Info("updateNodesStatus: " + node.Name)
 		nodeStatus := getNodeStatus(&node)
 		previousNodeStatus := getPreviousNodeStatus(node.Name, instance)
 
 		switch previousNodeStatus {
 		case "None":
+			log.Info("Previous status None: " + node.Name)
 			if nodeStatus == "SchedulingDisabled" {
 				// Node being removed
+				// Delete draining jobs
```

Comment:
is this ever executed? if the previous status was none, and the current one is schedulingDisabled, we did not created anything on the node yet, right?